src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
This "more general approach" needlessly allocates new object on the stack, while actually you only need to compare the `@ticks` ivar here.